packages/shared/src/components/squads/SharePostBar.tsx

```diff
@@ -121,8 +121,8 @@ function SharePostBar({
           name="share-post-bar"
           placeholder={`Enter URL${isMobile ? '' : ' / Choose from'}`}
           className={classNames(
-            'pl-1 tablet:min-w-[11rem] w-auto outline-none bg-theme-bg-transparent text-theme-label-primary focus:placeholder-theme-label-quaternary hover:placeholder-theme-label-primary typo-body',
-            url !== undefined && 'flex-1 pr-2',
+            'pl-1 tablet:min-w-[11rem] w-full flex-1 outline-none bg-theme-bg-transparent text-theme-label-primary focus:placeholder-theme-label-quaternary hover:placeholder-theme-label-primary typo-body',
```

Comment:
Can you check on desktop if this still provides support for reading history? If not, just apply the necessary styles for tablet sizes and laptop.